Revise (#65)
aaronpeikert/bayes-prereg: manuscript.Rmd

In any case a well written preregistration should aim to reduce the uncertainty about the theoretical risk and hence increase the persuasiveness of evidence.
A study with a perfect preregistration will, therefore, resemble the solid line in Figure \ref{fig:posterior}/\ref{fig:other-measures}.
Crucially, perfect means here that the theoretical risk can be judged with low uncertainty, not the the theoretical risk is necessarily high.

# Discussion

To summarize, we showed that higher theoretical risk and lower uncertainty about theoretical risk lead to higher expected epistemic value across a variety of measures.
The former result, that increasing theoretical risk leads to higher expected epistemic value, reconstructs the appeal of confirmatory research.
However, theoretical risk is something researcher have only limited control over, e.g., theories are often vague and illdefined, resources are limited, and increasing theoretical risk usually decreases detectability (a special instance of this trade off is the well known tension between alpha error and power).

> Before researchers may preregister a confirmatory study with high theoretical risk, they must ensure that they are able to conduct a confirmatory study.

Author: ```suggestion
While we believe that preregistration is always beneficial, it might be counterproductive to pursue high theoretical risk if the conditions for highly confirmatory research are not given.
```

Researchers must develop theories sufficiently and then have the resources to adequately balance detectability against theoretical risk.

> A preregistration may at most help to invest some time into developing more specific, hence riskier, implications of a theory.

Author: ```suggestion
In terms of preparing the conditions for confirmatory research, preregistration may at most help to invest some time into developing more specific, hence riskier, implications of a theory.
```

Otherwise the uncertainty about the theoretical risk will diminish the advantage of confirmatory research.
Second, even under less than ideal circumstances for confirmatory research, preregistration is beneficial.
Preregistering exploratory studies increases the expected epistemic value by virtue of reducing uncertainty about theorethical risk.
Nevertheless, exploratory studies will have a lower expected epistemic value than a more confirmatory study, if both are preregistered and have equal detectability.

Focusing on uncertainty reduction also explains two common practices of preregistration that do not align with a confirmatory research agenda.
First, researchers seldomly predict precise numerical outcomes, instead they use preregistrations to describe the process that generates the results.
Precise predictions would have very high theoretical risk (they are likely incorrect if the theory is wrong).
A statistical procedure, may have high or low theoretical risk depending on the specifics of the model used.
Specifying the process, therefore, is in line with the rational we propose here, but is incompatible with strictly confirmatory research.

> Second, researchers often have to deviate from the preregistration and make data-dependent decisions outside of the preregistration.

Reviewer: ```suggestion
Second, researchers often have to deviate from the preregistration and make data-dependent decisions after the preregistration.
```

If the only goal of preregistration is to ensure confirmatory research, such changes are not justifiable.
However, under our rational some changes may be justified.

> Any change would increase the uncertainty about the theoretical risk and may even decrease the theorethical risk.

Reviewer: ```suggestion
Any change increases the uncertainty about the theoretical risk and may even decrease the theorethical risk.
```

The changes still may be worthwhile, if the negative outcomes may be offset by an increase in detectability due to the change.

> Consider a preregistration that failed to specify what to do with missing values where the researchers subsequently encounter missings.

Reviewer: ```suggestion
Consider a preregistration that failed to specify how to handle missing values, and researchers subsequently encountering missing values.
```

> In such case, the detectability is zero, since the data cannot be analyzed without a decision about how to handle the missing data.

Reviewer: ```suggestion
In such case, detectability becomes zero because the data cannot be analyzed without a post-hoc decision about how to handle the missing data.
```

> Any such decision would constitute a deviation from the preregistration.

Reviewer: ```suggestion
Any such decision would constitute a deviation from the preregistration, which is perfectly waranted under our general objective.
```

> Furthermore, a reader cannot rule out that the researchers leveraged the decision to decrease theoretical risk, i.e., picking among all options the one that delivers the most beneficial results for the theory.

Reviewer: ```suggestion
Note that a reader cannot rule out that the researchers leveraged the decision to decrease theoretical risk, i.e., picking among all options the one that delivers the most beneficial results for the theory (in the previous example, chosing between various options of handling missing values).
```

Whatever decision they make, increased uncertainty about the theoretical risk is inevitable and the expected epistemic value is decreased compared to a world where they anticipated the need to deal with missing data.
However, it is still justified to deviate.
After all they have not anticipated the case and are left with a detectablilty of zero.
Any decision will increase detectability to a non-zero value, likely offsetting the increase in uncertainty.
The researchers also may do their best to argue that the deviation was not motivated by increasing theoretical risk, thereby, decreasing the uncertainty.

> Maybe there is one decision that fits well with the theory or with the study design.

Reviewer: ```suggestion
Ideally, there is a default decision that fits well with the theory or with the study design.
```

Or, if there is no obvious candidate, the researchers could conduct a multiverse analysis of the available options to deal with missings to show the influence of the decision [@steegenIncreasingTransparencyMultiverse2016].

> Using reduction in uncertainty as the objective for preregistration does not only explain some existing behavior, it allows to form recommendations to improve the practice of preregistration.

Reviewer: I don't understand "to explain some exisiting behavior".. of whom? and what exactly?

> Maybe most important, we now have a measure to differentiate a good preregistration from a bad one.

Reviewer: ```suggestion
Importantly, we now have a theoretical measure to gauge the utility of preregistrations.
```

> Using reduction in uncertainty as the objective for preregistration does not only explain some existing behavior, it allows to form recommendations to improve the practice of preregistration.

Author: ```suggestion
As explained above, reduction in uncertainty as the objective for preregistration does not only explain some existing behavior, that does not align with confirmation as a goal, it allows to form recommendations to improve the practice of preregistration.
```

In particular, a preregistration should be specific about the procedure that is intended to generate evidence for a theory.
Such procedure may accommodate a wide range of possible data, i.e., it may be exploratory.
The theoretical risk, however low, must be communicated clearly.
Parts of the process left unspecified, imply uncertainty, which is what a preregistration should reduce.

> However, specifying procedures that can be expected to not work, will lead to deviation, and uncertainty later.

Reviewer: ```suggestion
However, specifying procedures that can be expected to not work, will lead to deviation, and subsequently to larger uncertainty.
```

> We have proposed a highly specific, yet practical preregistration procedure called preregistration as code (PAC) elsewhere [@peikertReproducibleResearchTutorial2021].

Reviewer: ```suggestion
We have proposed a workflow for preregistration called preregistration as code (PAC) elsewhere [@peikertReproducibleResearchTutorial2021].
```

> In a PAC, researchers use computer code instead of a verbal description.

Reviewer: ```suggestion
In PAC, there is no distinction between a preregistration document and a final scientific report because the final report including all data analyses is created as a dynamic document including dynamically generated statistical analyses and natural language parts.
```

This enables a preregistrations to accommodate many different data patterns, yet is very specific about exactly how probable it is to generate evidence even when the theory does not hold (theoretical risk).
To reduce the liklihood of deviation, we suggest that a PAC is tested on simulated or pilot data.

> Another advantage is that with a PAC, deviations from the preregistration can be isolated and inspected.

Reviewer: ```suggestion
Another advantage is that with a PAC, deviations from the preregistration can be clearly (and if necessary, automatically) isolated, highlighted, and inspected.
```

> This is possible because in a PAC, preregistration and final manuscript are different versions of the same dynamic document, which can be compared via version control.

Reviewer: this needs to be explained at first in l.517!

However, while PAC is ideally suited to reduce uncertainty about theoretical risk, other more traditional forms of preregistration are also suited to achieve this goal.

Contrary to what is widely assumed about preregistration, a preregistration is not necessarily a seal of confirmatory research.

> Confirmatory research would be much less persuasive without preregistration, but in our view preregistration simply communicates the extend of confirmation, i.e., theoretical risk, of a study.

Reviewer: ```suggestion
Confirmatory research would often be less persuasive without preregistration, but in our view preregistration primarily communicates the extent of confirmation, i.e., theoretical risk, of a study.
```

Author: ```suggestion
Confirmatory research would almost always be less persuasive without preregistration, but in our view, preregistration primarily communicates the extent of confirmation, i.e., theoretical risk, of a study.
```
